integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/WebhookSiteSteps.java

```diff
@@ -32,4 +37,18 @@ public void webhookSiteContainsRequest(String requestText, int timeoutMinutes) {
                                 requestTextWithoutPlaceholders)
                         .anyMatch(requestContent -> requestContent.contains(requestTextWithoutPlaceholders)));
     }
+
+    @Then("^Webhook site does not contains request with text \"([^\"]*)\" within (\\d+) (second|seconds|minute|minutes)$")
+    public void webhookSiteDoesNotContainsRequest(String requestText, long timeoutAmount, String timeoutChronoUnits) throws InterruptedException {
+        String requestTextWithoutPlaceholders = ContextResolver.resolveWithScenarioContext(context, requestText);
+        ChronoUnit parsedTimeoutChronoUnits = ChronoUnitConverter.parseChronoUnits(timeoutChronoUnits);
+        Instant timeoutTime = Instant.now().plus(Duration.of(timeoutAmount, parsedTimeoutChronoUnits));
+        while (timeoutTime.isAfter(Instant.now())) {
+            TimeUnit.of(ChronoUnit.SECONDS).sleep(1);
+            assertThat(WebhookSiteResource.requests())
+                    .map(WebhookSiteRequest::getContent)
+                    .as("Checking that WebHook site doesn't contain request containing text: '%s'", requestTextWithoutPlaceholders)
+                    .noneMatch(requestContent -> requestContent.contains(requestTextWithoutPlaceholders));
+        }
```

Comment:
@Srihari1192 @sutaakar to me it seem to worth to try keep the method approaches very similar, as we already introduced awaitility dependency in this class, can't we implement the new method using it? That way we do not need while cycle with custom timeout simulation
```java
        String requestTextWithoutPlaceholders = ContextResolver.resolveWithScenarioContext(context, requestText);
        ChronoUnit parsedTimeoutChronoUnits = ChronoUnitConverter.parseChronoUnits(timeoutChronoUnits);
        
        Awaitility.await()
                .atLeast(Duration.of(timeoutAmount, parsedTimeoutChronoUnits))
                .pollInterval(Duration.ofSeconds(1))
                .untilAsserted(() -> assertThat(WebhookSiteResource.requests())
                    .map(WebhookSiteRequest::getContent)
                    .as("Checking that WebHook site doesn't contain request containing text: '%s'", requestTextWithoutPlaceholders)
                    .noneMatch(requestContent -> requestContent.contains(requestTextWithoutPlaceholders)));
```

Reply:
@sutaakar well, isn't that achieved by combination of : 

`atLeast` and `noneMatch`?

Reply:
`atLeast` has a different purpose: `Condition has to be evaluated not earlier than timeout before throwing a timeout exception.`
Here the condition is evaluated immediately, such implementation would throw an exception that the condition was evaluated sooner than time defined in `atLeast`.

Reply:
We need to make sure that the condition is not evaluated at all.

Reply:
I think I do not understand.

The documentation [1] says: In case the condition is fulfilled before the duration specified by atLeast an exception is thrown indicating that the condition shouldn't be completed earlier than the specified duration.

So in our case, we say check few seconds webhooks site, to be sure request with text hasn't appeared. Or?

ps. I would simply open PR with my proposal, to see the effect,  but CI is broken.

[1]
https://github.com/awaitility/awaitility/wiki/Usage#at-least

Reply:
> So in our case, we say check few seconds webhooks site, to be sure request with text hasn't appeared. Or?

Yes, exactly.